### ISSUE TITLE
Transition Complete, event is not passed 

### ISSUE THREAD

**Issue by ventura-eesc** [fomantic/Fomantic-UI#1018]
# Bug Report

TypeError: Cannot read property 'target' of undefined
    at Object.complete (semantic.js:23151)
    at Object.animate (semantic.js:23120)
    at Object.initialize (semantic.js:23025)
    at HTMLDivElement.<anonymous> (semantic.js:23949)
    at Function.each (jquery.js:354)
    at jQuery.fn.init.each (jquery.js:189)
    at jQuery.fn.init.$.fn.transition (semantic.js:22975)
    at Object.hide (semantic.js:5792)
    at HTMLDivElement.hide (semantic.js:5705)
    at Object.invoke (semantic.js:6113)

transition.js complete line : 217

```javascript
          if(event.target === element) {
              event.stopPropagation();
          }
```

This function is called from 
transition.js animate line : 187

```javascript
              module.debug('New animation started, completing previous early', settings.animation);
              instance.complete();

```
Not passing the event argument that we will get the error.

## How to reproduce:
Some times doing something like this:
            this.$pageDimmer.dimmer('show');
            this.$pageDimmer.dimmer('hide');

## Version
2.7.7

## To fix 
transition.js complete line : 217

```javascript
          if(event && event.target === element) {
              event.stopPropagation();
          }

```

**Comment by lubber-de** [comment]
That's already fixed since 2.7.6
Please check what FUI version you are using.

**Comment by lubber-de** [comment]
See here:
https://github.com/fomantic/Fomantic-UI/blob/4b5cd6bc0dd17cf2645d3a5f1564cfe20691b7a3/src/definitions/modules/transition.js#L218
